import-dashboards.py

```diff
@@ -242,6 +242,25 @@ def add_datasources(api_key):
             print ' * Cannot add PTSummary Data Source'
             sys.exit(-1)
 
+    if 'Prometheus AlertManager' not in ds:
+        print ' * Prometheus AlertManager Data Source'
+        data = json.dumps({
+            'name': 'Prometheus AlertManager',
+            'type': 'camptocamp-prometheus-alertmanager-datasource',
+            'url': 'http://localhost/alertmanager',
```

Comment:
One option is too reach Alertmanager directly, not via nginx

Reply:
Thank you for suggestions. Yes. It can be done in these ways. I have added a cookie name into plugin's Whitelisted Cookies list.

Reply:
I'm afraid we have to access Alertmanager directly, not via nginx. Otherwise, we hit pmm-managed several times more on each dashboard load